tests/Unit/ApparentHorizons/Test_ComputeHorizonVolumeQuantities.cpp

```diff
@@ -0,0 +1,476 @@
+      jacobian_target_to_inertial =
+          domain.blocks()[0].moving_mesh_grid_to_inertial_map().jacobian(
+              map_logical_to_grid(logical_coordinates(mesh)),
+              temporal_id.step_time().value(),
+              domain_creator->functions_of_time());
+      inv_jacobian_logical_to_target = inv_jacobian_logical_to_grid;
+      target_frame_coords = map_logical_to_grid(logical_coordinates(mesh));
+    } else {
+      static_assert(std::is_same_v<TargetFrame, Frame::Inertial>,
+                    "TargetFrame must be the Inertial frame");
+      inv_jacobian_logical_to_target = inv_jacobian_logical_to_inertial;
+      // jacobian_target_to_inertial is the identity in this case.
+      jacobian_target_to_inertial =
+          Jacobian<DataVector, 3, TargetFrame, Frame::Inertial>(
+              mesh.number_of_grid_points(), 0.0);
+      for (size_t i = 0; i < 3; ++i) {
+        jacobian_target_to_inertial.get(i, i) = 1.0;
+      }
+      target_frame_coords =
+          domain.blocks()[0].moving_mesh_grid_to_inertial_map()(
+              map_logical_to_grid(logical_coordinates(mesh)),
+              temporal_id.step_time().value(),
+              domain_creator->functions_of_time());
+    }
+  } else {
+    // time-independent.
+    static_assert(std::is_same_v<TargetFrame, Frame::Inertial>,
+                  "TargetFrame must be the Inertial frame");
+    // Don't need to define jacobian_target_to_inertial
+    ElementMap<3, Frame::Inertial> map_logical_to_inertial{
+        element_ids[0], domain.blocks()[0].stationary_map().get_clone()};
+    target_frame_coords = map_logical_to_inertial(logical_coordinates(mesh));
+    inv_jacobian_logical_to_inertial =
+        map_logical_to_inertial.inv_jacobian(logical_coordinates(mesh));
+    inv_jacobian_logical_to_target = inv_jacobian_logical_to_inertial;
+  }
+
+  // Set up analytic solution.
+  gr::Solutions::KerrSchild solution(1.0, {{0.1, 0.2, 0.3}},
+                                     {{0.03, 0.01, 0.02}});
+  const auto solution_vars_target_frame = solution.variables(
+      target_frame_coords, 0.0,
+      typename gr::Solutions::KerrSchild::tags<DataVector, TargetFrame>{});
+  const auto& lapse =
+      get<gr::Tags::Lapse<DataVector>>(solution_vars_target_frame);
+  const auto& dt_lapse =
+      get<Tags::dt<gr::Tags::Lapse<DataVector>>>(solution_vars_target_frame);
+  const auto& d_lapse = get<
+      typename gr::Solutions::KerrSchild::DerivLapse<DataVector, TargetFrame>>(
+      solution_vars_target_frame);
+  const auto& shift = get<gr::Tags::Shift<3, TargetFrame, DataVector>>(
+      solution_vars_target_frame);
+  const auto& d_shift = get<
+      typename gr::Solutions::KerrSchild::DerivShift<DataVector, TargetFrame>>(
+      solution_vars_target_frame);
+  const auto& dt_shift =
+      get<Tags::dt<gr::Tags::Shift<3, TargetFrame, DataVector>>>(
+          solution_vars_target_frame);
+  const auto& g = get<gr::Tags::SpatialMetric<3, TargetFrame, DataVector>>(
```

Comment:
I'd perfer a more verbose name like `spatial_metric` over the single-character variable name (and to a lesser extent, for the `dt_g` and `d_g` below)